### PR TITLE
fix(ustream): corrects the url for ustream

### PR DESCRIPTION
**Pull request by SnehaMidha**
This issue was not mentioned but the rendered link was not valid for ustream .This commit fixes that issue.

### CODE REVIEW

**Comment by ritz078** [comment]
I think it works good even now https://codepen.io/ritz078/pen/QpKQQy . Can you fork this and give an example where this doesn't work ? I am using v4.1.16.

**Comment by SnehaMidha** [comment]
Everything woks fine on codepen but on my system, using the file demo/index.html, this appears with old code:
image
It says that we are trying to access the link as http://www.http//www.ustream.tv/embed/1524, and it works fine only after changing that line.

**Comment by ritz078** [comment]
In that case can't add `//www` blindly in front. There should be a check whether its already present.

File: src/js/modules/template.js [hunk 1]
```diff
@@ -93,7 +93,14 @@ export default {
     if (match.indexOf('/embed/') < 0) {
       id.splice(1, 0, 'embed')
     }
-    return `<div class="ejs-embed ejs-ustream"><iframe src="//www.${id.join('/')}" height="${options.videoHeight}" width="${options.videoWidth}"></iframe></div>`
+	console.log("ustream");
```
Comment (ritz078) [inline src/js/modules/template.js]: remove this

File: src/js/modules/template.js [hunk 2]
```diff
@@ -93,7 +93,14 @@ export default {
...
+	console.log(id);
+    //check if '//www.' is there in the link
+    if(match.indexOf('/www\.ustream\.tv/') < 0){
+      return `<div class="ejs-embed ejs-ustream"><iframe src="//www.${id.join('/')}" height="${options.videoHeight}" width="${options.videoWidth}"></iframe></div>`
```
Comment (ritz078) [inline src/js/modules/template.js]: prevent multiple returns instead assign the src to a variable and use that in the template.

**Comment by ritz078** [comment]
Also I see test is breaking

**Comment by ritz078** [comment]
Have you tested with any example ?

**Pull request merged**